### PR TITLE
Fixed #27440 -- Allowed reverse() to construct an absolute URL.

### PR DESCRIPTION
**Pull request by mislavcimpersak**
trac issue: https://code.djangoproject.com/ticket/27440

### CODE REVIEW

File: docs/ref/urlresolvers.txt [hunk 1]
```diff
@@ -75,6 +75,10 @@ construct a fully qualified URL when using the method. For example::
     >>> reverse('admin:app_list', kwargs={'app_label': 'auth'}, request=request)
     'htttp://example.com/admin/auth/'
 
+.. versionchanged:: 1.11
+
+    The ``request``argument was added.
```
Comment (mr-bo-jangles) [inline docs/ref/urlresolvers.txt]: missing a space after the ``request`` part

**Comment by mr-bo-jangles** [comment]
Looking good :+1:

**Comment by mislavcimpersak** [comment]
I'm opening a discussion on django-developers group regarding this. Will link when it will be public.

File: django/urls/base.py [hunk 2]
```diff
@@ -88,7 +89,14 @@ def reverse(viewname, urlconf=None, args=None, kwargs=None, current_app=None):
         if ns_pattern:
             resolver = get_ns_resolver(ns_pattern, resolver)
 
-    return force_text(iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs)))
+    url = force_text(iri_to_uri(resolver._reverse_with_prefix(
```
Comment (sjoerdjob) [inline django/urls/base.py]: Why is the result of `iri_to_uri` stored in a variable named `url` instead of `uri`?

Comment (mislavcimpersak) [inline django/urls/base.py]: nice catch. changed it.

**Comment by mislavcimpersak** [comment]
Closing pull request as discussed on google-groups.

**Pull request closed without merging**